src/monattack.cpp

```diff
@@ -4699,7 +4690,8 @@ bool mattack::slimespring( monster *z )
             }
             if( player_character.has_effect( effect_bleed ) ) {
                 if( one_in( 2 ) ) {
-                    player_character.remove_effect( effect_bleed );
+                    effect e = player_character.get_effect( effect_bleed );
```

Comment:
```suggestion
                    effect &e = player_character.get_effect( effect_bleed );
```
Otherwise you would modify the local variable, which is going out of scope soon after and that would not make any sense.